.github/workflows/test.yml

```diff
@@ -28,8 +49,6 @@ jobs:
           POSTGRES_DB: odoo
         ports:
           - 5432:5432
-    env:
-      PIP_EXTRA_INDEX_URL: "https://wheelhouse.shopinvader.com/simple/"
```

Comment:
We need to preserve this.

Reply:
Could we do this at the same time of switching ?